src/TasklistFile.js

```diff
@@ -1,8 +1,6 @@
 import { update } from "lodash";
```

Comment:
As stated in the previous review, please remove this import as it is unnecessary.

Reply:
Kindly I used the Footer and header as Ribbons Asides achieving semantical and accessibility concerns